msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
```

Comment:
You can comment on the commit diffs themselves. I plan to just merge this without a Pull Request as soon as I have an `.msi` that works (read: installs Git and runs `post-install.bat` correctly).